src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,9 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter(user -> user.getName().equals(name))
```

Comment:
matchName 이라는 메서드를 호출해도 좋을 것 같아요
```suggestion
                .filter(user -> user.matchName(name))
```